test-infra/kfp/README.md

```diff
@@ -10,12 +10,6 @@
     make kfp-update
     ```
 
-1. Run:
-
-    ```bash
-    make hydrate-kfp-manifests
-    ```
-
     It generates raw k8s resources in acm-repos folder which is source of truth for the cluster via gitops.
```

Comment:
Can we keep this instruction? Instead of removing it, maybe we should give more guidance to users about how to use it.

From my opinion: We should run `make hydrate-kfp-manifests` first. It is important for editor to be able to review the generated manifests, before pushing it to the CI cluster. After validation, editor can run `make kfp-update` to apply the change.

Reply:
Updated. PTAL.